### PR TITLE
Community showcase, blueprints, new guides, & more 

### LINKED ISSUE #172: [DOC-87] Improve SEO
E.g., feed docs to LLM to get improved SEO results. Or, just do the ol fashioned way, manually, but improving keywords, tags, and cross page links.

<sub>From [SyncLinear.com](https://synclinear.com) | [DOC-87](https://linear.app/tableland/issue/DOC-87/improve-seo)</sub>

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary

Adds a number of new features:
- A (simple) showcase component that lists out partners & a brief on their use case, plus relevant links. Searchable/filterable by Tableland vs. Textile and chains/tags. The showcase includes DIMO, wxm, DR/VRS, KittyKart, Rigs, and the Studio.
- A new "features" page, modeled off of [Supabase](https://supabase.com/docs/guides/getting-started/features).
- New SQL blueprint walkthroughs for how to set up tables for gaming (state, leaderboard, inventory) and DataDAOs.
- Changed cost estimator component to take user input (it would have been useful in a convo with a dev re: costs).
- New guide on how to chunk queries to under 35kb in the SDK to help with large query use cases.
- New Local Tableland docs that describe the available methods & usage.
- FAQ pages for each client.
- Added simple link/callout to Go SDK.
- Minor SEO changes to hit on top CTR keywords in h1/h2 on landing page items and keywords (e.g., "web3 database" should be used as much as possible).

Closes #172, #89, #97, #92, #183, #98, #81, #96.

### CODE REVIEW

**Review by joewagner — COMMENTED** [review]
@dtbuchholz tons of really good docs here!  I'm still reading everything, I've been using the heart emojis to keep track of what I've reviewed you can ignore them.  
I have an inline suggestion no hurry discussing, but I figured I'd comment now since it's taking me a while to read everything.  I'll try to finish reviewing this today.

File: docs/sdk/walkthroughs/chunking-queries.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  - limits
+  - 35kb
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Overview
+
+Tableland create and write statements flow through a pass chain. Because all chains have execution limits, there is a [35kb limit](/fundamentals/limits) imposed on the size of a single transaction. This means that if you try to execute a statement that is larger than 35kb, it will fail. We'll walk through how you can implement your own query chunking logic to get around this limit and submit multiple transactions to execute a set of statements.
```
[the remark this answers is not in the document]
Comment (dtbuchholz) [reply docs/sdk/walkthroughs/chunking-queries.md]: good catch! (i shouldn't trust GH copilot finishing my sentences as much as i do lol)

**Comment by dtbuchholz** [comment]
> @dtbuchholz tons of really good docs here! I'm still reading everything, I've been using the heart emojis to keep track of what I've reviewed you can ignore them. I have an inline suggestion no hurry discussing, but I figured I'd comment now since it's taking me a while to read everything. I'll try to finish reviewing this today.

thanks! and fwiw, i test out all of the snippets i write. i also came across [cspell](https://www.npmjs.com/package/@cspell/eslint-plugin), which might be useful for spell checking since there's always a possibility of misspellings.

**Review by joewagner — APPROVED** [review]
Looks good!

**Comment by joewagner** [comment]
> thanks! and fwiw, i test out all of the snippets i write. i also came across [cspell](https://www.npmjs.com/package/@cspell/eslint-plugin), which might be useful for spell checking since there's always a possibility of misspellings.

good to know!  I have not been testing all the snippets, so I'm glad you are.  
A spell checker sounds good to me.

**Pull request merged**